fpgaconvnet/models/layers/ConvolutionLayer.py

```diff
@@ -106,13 +75,12 @@ def __init__(
             self.use_uram = False
         elif self.backend == "chisel":
             self.double_buffered = False
-            self.stream_weights = stream_weights
```

Comment:
I'm not sure why this is deleted?

Reply:
@AlexMontgomerie Any idea about this one?